docs/ref/models/fields.txt

```diff
@@ -771,7 +771,7 @@ A :class:`CharField` that checks that the value is a valid email address using
 ``FileField``
 -------------
 
-.. class:: FileField(upload_to=None, max_length=100, **options)
+.. class:: FileField(upload_to='', storage=None, **options)
```

Comment:
IMO, it's worth keeping `max_length` in docs as for `EmailField`, `URLField`, `FilePathField` etc.